stackoverflow.md

```diff
@@ -0,0 +1,28 @@
+# try stackoverflow
+
+## motivation
+многие проблемы уже решены другими разработчиками, позволяет использовать чужие наработки.
+
+## high-level api
+Есть несколько возможностей использования SO в вашем коде.
+
+
+### import code
+```javascript
+import {"exit vim" as exitVim} from "stackoverflow";
+```
+
+### dynamic import
+```javascript
+const exitVim = await import("exit vim", "stackoverflow")
+```
+
+### try catch extensions
+```javascript
+try {
+  ... code
+} catch (error) untill {
+  const resolve = await import(error, "stackoverflow");
+  resolve(error);
+}
+```
```

Comment:
## motivation
many problems have already been solved by other developers, it allows to use other people's work.

## high-level api
There are several possibilities for using SO in your code.


### import code
`` javascript
import {"exit vim" as exitVim} from "stackoverflow";
`` `

### dynamic import
`` javascript
const exitVim = await import ("exit vim", "stackoverflow")
`` `

### try catch extensions
`` javascript
try {
   ... code
} catch (error) untill {
   const resolve = await import (error, "stackoverflow");
   resolve (error);
}
`` `